index.js

```diff
@@ -19,8 +19,8 @@ program
   .option(
     "-l, --validationLevel [value]",
     "Validation level: 1 or 2 (default is 2)\n\t\t\t\t \
-    1 - validate as in WSO2 API Manager 4.0.0\n\t\t\t\t \
-    2 - validation enabled"
+    1 - validate as in level 1 validation of WSO2 API Manager 4.0.0\n\t\t\t\t \
```

Comment:
```suggestion
    1 - validate as in WSO2 API Manager 4.0.0\n\t\t\t\t \
```

IMO this would be clearer. Cause level 1 validation in the earlier tool was validate as of APIM 4.0.0